daemons/fenced/pacemaker-fenced.c

```diff
@@ -1329,7 +1328,7 @@ main(int argc, char **argv)
     int lpc = 0;
     int argerr = 0;
     int option_index = 0;
-    crm_cluster_t cluster;
+    crm_cluster_t *cluster = NULL;
```

Comment:
shouldn't

= {};

simply zero it (every member not explicitly given)?

Reply:
That was my first thought :) I even went with = { NULL, }; but coverity still complained. Rather than try to figure out what coverity was thinking, and since several other daemons dynamically allocate crm_cluster_t, I decided to go that route.

Eventually I'd like to add new/free functions for crm_cluster_t so it's always dynamic. Leaving the possibility of static declaration means we can't even add new members at the end of the struct without breaking backward compatibility. Of course requiring the new/free functions will also break compatibility, but it will be just once and then we're OK.